CMakeLists.txt

```diff
@@ -41,13 +41,9 @@ if( NOT Cabana_ENABLE_MPI )
   message( FATAL_ERROR "Cabana must be compiled with MPI" )
 endif()
 
-option(CabanaMD_REQUIRE_SERIAL "Build CabanaMD with Serial support" ON)
-option(CabanaMD_REQUIRE_OPENMP "Build CabanaMD with OpenMP support" OFF)
-option(CabanaMD_REQUIRE_CUDA "Build CabanaMD with Cuda support" OFF)
-option(CabanaMD_REQUIRE_HIP "Build CabanaMD with HIP support" OFF)
-
 set(CABANAMD_SUPPORTED_DEVICES SERIAL OPENMP CUDA HIP)
 foreach(_device ${CABANAMD_SUPPORTED_DEVICES})
+  option(CabanaMD_REQUIRE_${_device} "Build CabanaMD with ${_device} support" ${Kokkos_ENABLE_${_device}})
```

Comment:
Why did you call it "require"?  It feels like it should be "enable".

Reply:
I was just following Cabana